src/framework/entity/entity_entity.js

```diff
@@ -229,6 +273,13 @@ pc.extend(pc, function () {
             this.c[name].enabled = false;
         }
 
+        //Stop any coroutines
+        if (this._coroutines) {
+            this._coroutines.forEach(function (c) {
```

Comment:
`Array.forEach` does look semantically well, but most benefit it has over normal loop is to have scoped local variable. In this case it is not necessary, and classic for loop is way more efficient in this case.